ui-speedspacechart/src/components/common/SettingsPanel.tsx

```diff
@@ -26,6 +34,8 @@ const SettingsPanel = ({ color, store, setStore, translations }: SettingsPanelPr
       id="settings-panel"
       style={{ background: `rgba(${color.substring(4, color.length - 1)}, 0.4)` }}
       className="font-sans"
+      onMouseMove={() => setShowReticleLayer(false)}
```

Comment:
Why not use onMouseEnter ?

Reply:
For the first case (the cursor is near the panel), it has been validated by @thibautsailly  to leave it like this

Reply:
for the first comment: the two cases are ok for me, I can adapt the position of the box if you want, let me know if you want me to modify it 
for the PR layers: my bad, I just missed a pixel

Reply:
Yes it would be nice to adapt the position of the box if we are close to the settings panel :+1:

Reply:
done